client/java/src/main/java/io/openlineage/client/transports/HttpTransport.java

```diff
@@ -33,6 +34,7 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.util.EntityUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
is this `org.jetbrains.annotations.NotNull` a `NotNull` you wanted to import?

Reply:
nooooo, thanks for noticing